### ISSUE TITLE
Update organizer list

### ISSUE THREAD

**Issue by Thrillberg** [Trianglerb/trianglerb.github.io#10]
Add me to organizer list at bottom of page 😬
Does anyone want to be removed from the list? Does @drapergeek want to be added to the list?

**Comment by drapergeek** [comment]
> Does @drapergeek want to be added to the list?

No thank you, will try to help with things here and there but I'm unwilling to commit to anything else in life at the moment 😄

**Comment by Thrillberg** [comment]
Ok, understood! @base10 do you want to remain on the list of organizers on the website (and on the Meetup group)? I'll ask Brandon and TJ separately (on Slack).

**Comment by BrandonMathis** [comment]
@Thrillberg I think we can drop TJ from the origanizers list and I have no objection to you adding yourself. Please make a PR fixing and referencing this issue and then we can close this out.

**Comment by base10** [comment]
Please keep me for now, although we ought to get some meetings together :D

**Comment by Thrillberg** [comment]
>we ought to get some meetings together

Agreed! We've got one West End Ruby on the calendar at the moment but nothing else. Want to organize something else? 😃 😃 😃
